src/activities/menu/Menu.qml

```diff
@@ -122,6 +128,9 @@ ActivityBase {
         id: background
         source: menuActivity.url + "background.svg"
         sourceSize.width: parent.width
+        height: menuActivity.currentTag === "search" && ApplicationSettings.isVirtualKeyboard ?
+                    main.height - (((keyboard.keyHeight+keyboard.rowSpacing) * keyboard.rows)+keyboard.margin)
```

Comment:
getting the keyboard.height is not the same? If not, there should be a property in it to get the value, it should not be computed outside